server/lib/csv.js

```diff
@@ -56,10 +64,15 @@ export const getCollectiveTransactionsCsv = async (collective, { startDate, endD
   return getTransactionsCsv('transactions', collective, { startDate, endDate, kind });
 };
 
-export const getHostTransactionsCsv = async (collective, { startDate, endDate } = {}) => {
+export const getHostTransactionsCsvAsAdmin = async (collective, { startDate, endDate } = {}) => {
   if (parseToBoolean(config.restService.fetchHostTransactionsCsv) === false) {
     return;
   }
 
-  return getTransactionsCsv('hostTransactions', collective, { startDate, endDate });
+  return getTransactionsCsv('hostTransactions', collective, {
+    startDate,
+    endDate,
+    useAdminAccount: true,
+    add: ['accountName', 'oppositeAccountName'],
```

Comment:
I would prefer to have it as defaults on the REST service side. So there is only 1 default format we  promote and spread.

Reply:
I moved that to https://github.com/opencollective/opencollective-rest/pull/390/files#diff-275afb3c816085cf369751b24d6b5354d4fc2d4fa194d6a979a9fe1692ca0115R272